lib/_emerge/depgraph.py

```diff
@@ -11385,7 +11387,7 @@ def _spinner_stop(spinner):
     portage.writemsg_stdout(f"Dependency resolution took {darkgreen(time_fmt)} s.\n\n")
 
 
-def backtrack_depgraph(settings, trees, myopts, myparams, myaction, myfiles, spinner):
+def backtrack_depgraph(settings: portage.package.ebuild.config.config, trees: dict, myopts: dict, myparams: dict, myaction, myfiles: [str], spinner: stdout_spinner) -> (bool, depgraph, [str]):
```

Comment:
You should be able to quote the type to avoid an import here:
```suggestion
def backtrack_depgraph(settings: "portage.package.ebuild.config.config", trees: dict, myopts: dict, myparams: dict, myaction, myfiles: [str], spinner: stdout_spinner) -> (bool, depgraph, [str]):
```

Also, when you run mypy before & afterwards, do you get many more errors?

Reply:
Quotes to avoid import is very useful I guess. So, we can remove the import of stdout_spinner and use quotes again. 
`mypy` says "Found 115 errors in 43 files (checked 1 source file)", when checking depgraph.py.

Reply:
How many errors did it give _before_ your change, vs _after_?

Reply:
I did not run `mypy` before. Ran after the commit and it gives 115 errors.

Reply:
I did not run `mypy` before. After the commit, it says 115 errors. I ran `mypy` now without the commit, it says "Found 111 errors in 42 files (checked 1 source file)".

Reply:
Ok, but run it before _now_ and compare?

Reply:
This is the output of `mypy depgraph.py` before the commit. https://dpaste.com/2X6BTB24X
This is the output of `mypy depgraph.py` after the commit. https://dpaste.com/BK8MNCWFN

Reply:
Okay, so this introduces 4 new mypy errors. Specifically:
```

lib/_emerge/depgraph.py:11390: error: Bracketed expression "[...]" is not valid as a type  [valid-type]
lib/_emerge/depgraph.py:11390: note: Did you mean "List[...]"?
lib/_emerge/depgraph.py:11390: error: Syntax error in type annotation  [syntax]
lib/_emerge/depgraph.py:11390: note: Suggestion: Use Tuple[T1, ..., Tn] instead of (T1, ..., Tn)